lib/css/components/_stacks-modals.less

```diff
@@ -48,7 +48,6 @@
 .s-modal--dialog {
     overflow-y: auto;
     visibility: hidden;
-    position: fixed; // We shouldn't NEED this, since the parent is fixed, but IE Edge needs it to properly scroll at small viewport sizes.
```

Comment:
You added this in 97d1b1d6ad299a296ac3cd3aa3751ca2803b03f3 &ndash; do you happen to recall what precisely the scrolling issue was that you where referring to at the time?

Reply:
No, I don't remember, and I can't reproduce what the comment seems to be referring to 🤔. Is it possible that Edge has been patched in the meantime?